src/ServicioAlCliente/Promocion.java

```diff
@@ -0,0 +1,59 @@
+package ServicioAlCliente;
+
+import Registro.Cliente;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Promocion {
+
+    public static void canjearPremio(Premios premio,Cliente cliente){
+
+        switch (premio.getTipo()) {
+            case ("boleto"):
+
+                if (cliente.getPuntosTotales() >= premio.getPuntosPremio() && premio.equals(Premios.PRIMERAOPCION)) {
```

Comment:
```suggestion
                if (clientePuntosTotales >= puntosPremio && premio.equals(Premios.PRIMERAOPCION)){
```
y asi en todos los otros condicionales